Actors/VisualObjects/VisualObjects.ActorService/ActorEventSource.cs

```diff
@@ -86,16 +88,16 @@ public void ActorMessage(Actor actor, string message, params object[] args)
             }
         }
 
-        [Event(ActorHostInitializationFailedEventId, Level = EventLevel.Error, Message = "Actor host initialization failed",
-            Keywords = Keywords.HostInitialization)]
-        public void ActorHostInitializationFailed(string exception)
-        {
-            this.WriteEvent(ActorHostInitializationFailedEventId, exception);
-        }
-
+        // For very high-frequency events it might be advantageous to raise events using WriteEventCore API.
+        // This results in more efficient parameter handling, but requires explicit allocation of EventData structure and unsafe code.
+        // To enable this code path, define UNSAFE conditional compilation symbol and turn on unsafe code support in project properties.

```

Comment:
If the high-frequency event code is not used can we please remove to from the event sources to clean up the sample?